.github/workflows/coverage.yml

```diff
@@ -26,6 +26,6 @@ jobs:
       run: cd build && ctest --output-on-failure
 
     - name: Upload report
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v3
       with:
```

Comment:
Hi. what's changed in version 3?

Reply:
As of February 1, 2022, v1 has been fully sunset and no longer functions

Reply:
Okey. Good!